BaseFilter/sections/antiadblock.txt

```diff
@@ -7780,6 +7780,7 @@ roya.tv#$##wrapfabtest { height: 1px!important; }
 findandfound.ga#%#//scriptlet("abort-on-property-read", "adb_warning")
 ||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,domain=findandfound.ga
 ! https://github.com/AdguardTeam/AdguardFilters/issues/56233
+!+ NOT_OPTIMIZED
 @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=breatheheavy.com
 ||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$redirect=googlesyndication-adsbygoogle,domain=breatheheavy.com,important

```

Comment:
Yes, it is more complicated. I'm not sure if the developers had time to study the scriptlets.
But can be used instead of the redirect rule.
@artemiv4nov

Reply:
So, should I replace a `@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=breatheheavy.com` rule to a newly suggested by @AdamWr, or we can keep it as is?

Reply:
As far as I understand, you can replace `||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$redirect=googlesyndication-adsbygoogle,domain=breatheheavy.com,important` by a scriptlet - `breatheheavy.com#%#//scriptlet('prevent-fetch', 'pagead2.googlesyndication.com')`

Reply:
Done, please re-check